python/oneflow/test/misc/test_autograd_functional.py

```diff
@@ -31,15 +32,37 @@ def _func_scalar(x):
         inputs = torch.randn(5, 5)
         v = torch.randn(5)
         result_tensor = torch.autograd.functional.vjp(_func_tensor, inputs, v)
-        result_scalar = torch.autograd.functional.vjp(_func_scalar, inputs)
+        #result_scalar = torch.autograd.functional.vjp(_func_scalar, inputs)
```

Comment:
这里有一些细节要注意：
- 首先，这里代码注释掉要写明原因（这个测试是否跑不了？是的话，有啥问题跑不了？什么问题修复之后可以取消注释？），其次如果这里有 bug 的话，可以整理出来最小复现的代码，然后提交一个 issue，并且把 issue 地址贴到原因中
```suggestion
        # TODO: autograd.grad interface has a bug here, uncomment when issue xx is fixed
        # result_scalar = torch.autograd.functional.vjp(_func_scalar, inputs)
```

Reply:
已上传issue并添加相应注释